engine/flink/management/periodic/src/test/scala/pl/touk/nussknacker/engine/management/periodic/db/InMemPeriodicProcessesRepository.scala

```diff
@@ -54,12 +55,12 @@ class InMemPeriodicProcessesRepository extends PeriodicProcessesRepository {
       id = id,
       processName = processName,
       processVersionId = VersionId.initialVersionId,
-      processJson = ScenarioParser.toGraphProcess(
+      processJson = ScenarioParser.toJson(
         EspProcessBuilder
           .id(processName.value)
           .source("start", "source")
           .emptySink("end", "KafkaSink")
-      ).marshalled,
+      ).noSpaces,
```

Comment:
Why noSpaces?

Reply:
Because of performance - we talked about, resolving